views/partials/beerpartial.hbs

```diff
@@ -0,0 +1,16 @@
+<section>
+ <!--{{#each beers}}-->
+ <div>
+    <img src="{{image_url}}" alt="beer name: {{name}}" class="beer-image">
+ </div>
+<div>
+   <a href="/beers/beer-i3f4d34s34b">{{name}}</a>
```

Comment:
This id, `i3f4d34s34b `, was just an example. You should've created the route for a detailed beer, a view for it and used the other method the API gives you for it to work. And here, you would pass the params you passed on the route. We'll learn more about that this week, and if you want, try doing it again after it!